.travis.yml

```diff
@@ -13,6 +13,7 @@ sudo: false
 branches:
```

Comment:
why not remove this completely so that it builds for everything upstream by default. Then if we ever have a brnach we dont want to build we can do something explicit then ?

Same for appveyor below.

But do check this (not specifying anything) works

Reply:
agreed :)